packages/fx-core/templates/plugins/resource/simpleauth/bicep/simpleAuthProvision.template.v2.bicep

```diff
@@ -1,5 +1,5 @@
 param provisionParameters object
```

Comment:
please help add `@secure()` decorator for provisionParameters, since it contains secrets

Reply:
added